app/src/main/java/com/cornellappdev/transit/ui/screens/HomeScreen.kt

```diff
@@ -114,7 +114,7 @@ fun HomeScreen(
 
     // Collect flow of rides through API
     val stopsApiResponse = homeViewModel.stopFlow.collectAsState().value
-    val queryResponse = homeViewModel.queryFlow.collectAsState().value
+    val queryResponse = homeViewModel.queryFlow.collectAsState(initial = emptyList()).value
```

Comment:
...and then from my other comment, you shouldn't need to supply an initial value here. Instead, the viewmodel specifies the initial value, which I think applies abstraction better.